packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -254,16 +254,22 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
   /** 
   Sign and submit an entry (not bcs serialized) transaction type to chain.
   @param transaction a non-bcs serialized transaction
+  @param options max_gas_amount and gas_unit_limit
   @return response from the wallet's signAndSubmitTransaction function
   @throws WalletSignAndSubmitMessageError
   */
   async signAndSubmitTransaction(
-    transaction: Types.TransactionPayload
+    transaction: Types.TransactionPayload,
+    options?: {
```

Comment:
after creating the TransactionOptions type in the core package (see other comment), we can
```suggestion
    options?: TransactionOptions
```

Reply:
commit [8230ee7](https://github.com/aptos-labs/aptos-wallet-adapter/pull/156/commits/8230ee71c9ad6a41e67bcd6a4d5c451125ab4d45)